modules/ROOT/pages/values-and-types/property-structural-constructed.adoc

```diff
@@ -88,15 +87,21 @@ Some of these types are not yet supported in either type predicate expressions o
 | `LOCAL DATETIME` | `TIMESTAMP WITHOUT TIMEZONE`
 | `LOCAL TIME` | `TIME WITHOUT TIMEZONE`
 | `MAP` |
-| `NODE` | `VERTEX`
+| `NODE` | `ANY NODE`, `VERTEX`, `ANY VERTEX`
+| `NOTHING` |
+| `NULL` |
 | `PATH` |
 | `POINT` |
-| `RELATIONSHIP` | `EDGE`
+| `PROPERTY VALUE` | `ANY PROPERTY VALUE`
+| `RELATIONSHIP` | `ANY RELATIONSHIP`, `EDGE`, `ANY EDGE`
 | `STRING` | `VARCHAR`
 | `ZONED DATETIME` | `TIMESTAMP WITH TIMEZONE`
 | `ZONED TIME` | `TIME WITH TIMEZONE`
 |===
 
+All Cypher Types contain the `null` value, in order to make them not nullable `NOT NULL` can be appended to the end of the type.
+e.g `BOOLEAN NOT NULL`, `LIST<FLOAT NOT NULL>`
```

Comment:
this should probably be a full sentence and not one sentence and a free hanging part :P